spk/squidguard/src/etc/squidclamav.conf.tpl

```diff
@@ -14,7 +14,7 @@
 maxsize 5000000
 
```

Comment:
i don't rename the file : 
in the installer i have add this line : 
sed "s/==HOSTNAME==/$hostname/g" ${ETC_DIR}/squidclamav.conf.tpl > ${ETC_DIR}/squidclamav.conf

for correct the path of the file when a virus is found ;)